src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -114,13 +92,44 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
   }
 
   /**
+   * Gets the current account.
+   *
+   * @return AccountInterface
+   */
+  public function getAccount() {
```

Comment:
This is actually something that should be moved to ER itself, being able to pass the account - don't you think?